src/components/configuration/ActionLinkConfig.vue

```diff
@@ -46,10 +45,24 @@
         />
 
         <div class="mt-4">
-          <p class="text-caption">
-            The action will be called whenever any of the selected variables change, respecting the minimum interval
-            between consecutive calls.
-          </p>
+          <p class="text-subtitle-1 font-weight-bold mb-2">Time-based:</p>
+          <v-radio-group v-model="dialog.autoRunType" class="mt-2">
+            <v-radio value="once" label="On startup"></v-radio>
+            <v-radio value="interval" label="Repeatedly"></v-radio>
+            <v-radio value="none" label="No time trigger"></v-radio>
+          </v-radio-group>
```

Comment:
Could this be a horizontal row, in order of frequency (e.g. `None | On startup | Repeatedly`)?
That seems more intuitive than the current ordering/placement. 

Ideally it would be a `v-slider`, with the [options as tick labels](https://vuetifyjs.com/en/components/sliders/#ticks), in which case the display condition for the text field could be `frequencySlider > 0`.

Reply:
Can we keep as it is in the current state? The user can slider for easy setting or type what they want for custom values. All edge-cases are already treated.

Also, I personally don't think the slider for the none/once/intervaled is very intuitive, and it also asks for a drag when a click would be enough.